index.html

```diff
@@ -38,11 +38,11 @@ <h1 class="coverCaption">Andrew<br /><span class="coverCaptionSmaller"><span sty
     </header>
     <nav>
         <ul>
-            <li><span class="navListItem"><a href="#"><img src="img/home_icon_c.png" class="navIcon" alt="home symbol" tabindex="1" /></a><span class="navIconCaption">home</span></span></li>
-            <li><span class="navListItem"><a href="#bio"><img src="img/bio_icon.png" class="navIcon" alt="heart symbol" tabindex="1" /></a><span class="navIconCaption">bio</span></span></li>
-            <li><span class="navListItem"><a href="#chef"><img src="img/chef_icon.png" class="navIcon" alt="chef symbol" tabindex="1" /></a><span class="navIconCaption">chef</span></span></li>
-            <li><span class="navListItem"><a href="#gallery"><img src="img/gallery_icon.png" class="navIcon" alt="gallery symbol" tabindex="1" /></a><span class="navIconCaption">gallery</span></span></li>
-            <li><span class="navListItem"><a href="#recipes"><img src="img/recipes_icon.png" class="navIcon" class="noPaddingBottom" alt="recipes symbol" tabindex="1" /></a><span class="navIconCaption">recipes</span></span></li>
+            <li><span class="navListItem"><a href="#" tabindex="1"><img src="img/home_icon_c.png" class="navIcon" alt="home symbol" /></a><span class="navIconCaption">home</span></span></li>
```

Comment:
Why do you use `tabindex"1"` everywhere?

Reply:
Should I use it only once at <ul> or <nav>?